src/encoded/tests/test_schema_analyte.py

```diff
@@ -31,3 +34,128 @@ def test_rna_integrity_requirements(
         "3932c9d7-c5e6-46c3-9e67-2ccd276f4b74",
         status=expected_status,
     )
+
+
+@pytest.mark.workbook
+@pytest.mark.parametrize(
+    "patch_body,expected_status",
+    [
+        ({"dna_integrity_number": 6}, 422),
+        ({"dna_integrity_number_instrument": "Agilent 4200 TapeStation"}, 422),
+        (
+            {
+                "dna_integrity_number": 6,
+                "dna_integrity_number_instrument": "Agilent 4200 TapeStation",
+            },
+            200,
+        ),
+    ],
+)
+def test_dna_integrity_requirements(
+    patch_body: Dict[str, Any],
+    expected_status: int,
+    es_testapp: TestApp,
+    workbook: None,
+) -> None:
+    """Ensure mutual requirements for DIN and instrument."""
+    patch_item(
+        es_testapp,
+        patch_body,
+        "3932c9d7-c5e6-46c3-9e67-2ccd276f4b74",
```

Comment:
I know you're following the example above here, but using `get_insert_identifier_for_item_type` as in `test_schema_library.py` is the better approach, since someone may change the UUID here and break all these tests.